tests/foreman/cli/test_environment.py

```diff
@@ -242,18 +241,13 @@ def test_positive_sc_params_by_id(self):
 
         @Assert: The command runs without raising an error
 
-        @BZ: 1219934
-
         """
         environment = make_environment()
         Environment.sc_params({
             'environment-id': environment['id'],
         })
-        if not bz_bug_is_open(1219934):
-            self.fail('BZ #1219934 is closed, should assert the content')
```

Comment:
It seems that the sc_params is not implemented as it should have

```python
return cls.execute(cls._construct_command(options), output_format='csv')
```

Asking for CSV output will make Robottelo parse and return it as a list of dicts which should be much simpler to deal with.